index.html

```diff
@@ -36,34 +37,106 @@ <h2>Software</h2>
         </thead>
         <tbody>
           <tr>
-            <td>AIFF-hul</td>
+            <td><a href="/modules/aiff/" hreflang="en" rel="help">AIFF-hul</td>
             <td>Audio Interchange File Format</td>
             <td>.aif</td>
             <td>audio/x-aiff</td>
           </tr>
           <tr>
-            <td>ASCII-hul</td>
+            <td><a href="/modules/ascii/" hreflang="en" rel="help">ASCII-hul</td>
             <td>ASCII text</td>
-            <td>.txt</td>
-            <td>text/plain</td>
+            <td></td>
```

Comment:
These are the entries for the "BYTESTREAM" module below no, not ASCII? In truth, I'd drop the BYTESTREAM module altogether from here as it's only the "default" handler for files JHOVE can't identify.